src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -773,6 +782,18 @@ private void ParseHelper(string[] args)
 
                     _configurationName = args[i];
                 }
+                else if (MatchSwitch(switchKey, "debugpipename", "deb"))
+                {
+                    ++i;
+                    if (i >= args.Length)
+                    {
+                        WriteCommandLineError(
+                            CommandLineParameterParserStrings.MissingDebugPipeNameArgument);
+                        break;
+                    }
+
+                    _debugPipeName = args[i];
```

Comment:
There's a max length allowed for the pipename (I hit this on macOS when enabling this).  I have it as a comment in the code somewhere.  We should validate here otherwise it'll fail when creating the pipe.

Reply:
Oh right yes. I forgot about that. It's 104 on macOS, 108 on Linux. I'll address that

Reply:
fixed.